.github/workflows/docs.yaml

```
@@ -0,0 +1,50 @@
+name: Docs
+
+on:
+    push:
+        branches: [main]
+    release:
+        types: [released]
+
+jobs:
+    docs:
+        runs-on: ubuntu-latest
+
+        steps:
+        - uses: actions/checkout@v2
+
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+              python-version: 3.8
+
+        - name: Install Dependencies
+          run: pip install .[doc]
+
+        - name: Build HTML artifact
+          run: "python release_docs.py"
+
+        - name: Upload HTML artifact
+          uses: actions/upload-artifact@v1
+          with:
+              name: DocumentationHTML
+              path: docs/_build/
+
+        # TODO: branch-specific folders for branches other than `main`
+        - name: Commit and publish documentation changes to gh-pages branch
+          run: |
+              git clone https://github.com/lost-theory/eip712 --branch gh-pages --single-branch gh-pages
```

Should it be like this?
```suggestion
              git clone https://github.com/ApeWorX/eip712 --branch gh-pages --single-branch gh-pages
```

---

Yes good catch! Updated it.